tests/functional/object/mcg/test_noobaa_metrics.py

```
@@ -218,3 +218,175 @@ def test_noobaa_space_available_using_cli(
+
+        # 8. Remove max-size quota (set to 0 = unlimited), verify writes succeed
+        logger.info("Removing max-size quota (setting to 0)")
+        mcg_obj.exec_mcg_cmd(
+            cmd=f"bucket update --max-size=0 {bucket_name}",
+            namespace=config.ENV_DATA["cluster_namespace"],
+            use_yes=True,
+        )
+        time.sleep(180)
+
+        quota_status = get_bucket_status_value(mcg_obj, bucket_name, "QuotaStatus")
+        logger.info(f"Quota status after removing max-size: {quota_status}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)